victron_ble/devices/smart_lithium.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from construct import Array, BitsInteger, BitStruct, ByteSwapped, Padding
+
+from victron_ble.devices.base import Device, DeviceData
+
+
```

Comment:
```suggestion
from enum Import Enum

class BalancerStatus(Enum):
  UNKNOWN = 0
  BALANCED = 1
  IMBALANCE = 3
  
```
Since we know the meaning of the balancer status values, would be nice to make it an enum.